### ISSUE TITLE
[3.2] Error on Entry Save

### ISSUE THREAD

**Issue by tobyjug** [craftcms/cms#4421]
### Description
When saving a new entry on 3.2 Beta 3 I get the following error:

```
Getting unknown property: craft\elements\Entry::creatorId
```

### Steps to reproduce

1. Create an entry
2. Click 'Save entry'

### Additional info

- Craft version: 3.2.0-beta.3
- PHP version: 7.2.14
- Database driver & version: MySQL 5.7.25
- Plugins & versions:
Super Table 2.1.20
Contact Form 2.2.5

**Comment by brandonkelly** [comment]
That’s been fixed for the next release.

To get the fix early, change your `craftcms/cms` requirement in composer.json to:

```json
"require": {
  "craftcms/cms": "3.2.x-develop#49acf4cc67bcc6ffefc1ed9cc3c722f2f6af1f0d as 3.2.0-beta.3",
  "...": "..."
}
```

Then run `composer update`.

**Comment by JayBox325** [comment]
Hi Brandon, thanks for all the quick fixes to 3.2 bugs. I've tried to install that version linked above, but I'm getting the following response from Composer:

```
Could not parse version constraint 3.2.x-develop#49acf4cc67bcc6ffefc1ed9cc3c722f2f6af1f0d as 3.2.0-beta.3: Invalid version string "3.2.x-develop#49acf4cc67bcc6ffefc1ed9 cc3c722f2f6af1f0d" in "3.2.x-develop#49acf4cc67bcc6ffefc1ed9cc3c722f2f6af1f0d as 3.2.0-beta.3", the alias source must be an exact version, if it is a branch name you should prefix it with dev-  
```

I'm not sure if this is expected or I'm biffing it. But tbf, there'll probably be a release very shortly anyway.

**Comment by brandonkelly** [comment]
Sorry! `-develop` should have been `-dev`. Just edited my previous comment.
